src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,46 @@
+package racingcar;
+
+public class Car {
+
+    String name;
+    int count;
+
+    Car(String name) {
+        this.name = name;
+    }
+
+    Car(String name, int ount) {
+        this.name = name;
+        this.count = count;
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getCount() {
+        return count;
+    }
+
+    public void setCount(int count) {
+        this.count = count;
+    }
+
+    void addCount() {
+        this.count++;
+    }
+
+    public String getCountPrint() {
+        String returnStr = "";
+        for(int i=0; i<this.count; i++) {
```

Comment:
부호 앞뒤엔 한 칸의 스페이스를 넣어주세요.
```suggestion
        for(int i = 0; i < this.count; i++) {